diagrams/config-manager.puml

```diff
@@ -0,0 +1,64 @@
+@startuml
+'DevCycle Config Manager
+group Config Manager
+group Constructor/Initialization
+start
+    if (options.UseRealtimeUpdates) then (yes)
```

Comment:
this option is named wrong here

Reply:
It's more "human" here - but if we want to make it exact that works too

Reply:
I was trying to make it clear that the value is being pulled from the options passed into the initialization call - thoughts on how to better do that? Our options aren't named consistently

Reply:
they are named consistently in most cases. Any case where they aren't should be fixed

Reply:
The fact you said "most" means they aren't :P  Plus - all server SDKs don't actually have this functionality enabled - so for the 1/1 for go - thoughts on using the `options.EnableRealtimeUpdates` ? or `options.DisableRealtimeUpdates` ? -
Go will be disabled by default - so will have Enabled as the option name

Reply:
it should be named the same as the existing option in the other SDKs

Reply:
and it should be enabled by default

Reply:
Until we're confident in it with the test harness - it's going to be disabled by default - cc @jonathannorris

The other SDKs this is relevant to (server sdks) do not have a current option - do we want to match the client sdks?

Reply:
yes, we want it to match the client SDKs